config/src/main/java/org/hyperledger/besu/config/GenesisConfigOptions.java

```diff
@@ -94,7 +94,9 @@ default boolean isConsensusMigration() {
 
   OptionalLong getCancunTime();
 
-  OptionalLong getShandongBlockNumber();
+  OptionalLong getFutureEipsTime();
+
+  OptionalLong getExperimentalEipsTime();
```

Comment:
ah ok, so experimental is intended to be for EIPs not as a general use feature flag? Makes sense I think, any non-EIP experimental feature probably warrants its own flag.